package-lock.json

```diff
@@ -38366,11 +38399,6 @@
 				}
 			}
 		},
-		"is-plain-obj": {
-			"version": "4.1.0",
-			"resolved": "https://registry.npmjs.org/is-plain-obj/-/is-plain-obj-4.1.0.tgz",
-			"integrity": "sha512-+Pgi+vMuUNkJyExiMBt5IlFoMyKnr5zhJ4Uspz58WOhBF5QoIZkFyNHIbBAtHwzVAgk5RtndVNsDRN61/mmDqg=="
-		},
 		"is-plain-object": {
```

Comment:
It would be great to bump the top-level `is-plain-object` so it gets hoisted for all packages.

Reply:
Nice find. Let me take care of that

Reply:
Hm, we don't declare a top-level `is-plain-object`, this one is a dependency of other packages - are you suggesting that we add it specifically as a top-level dependency for the project?

Reply:
No, you can do it temporarily just to hoist all npm dependencies used.

Reply:
Actually, the opposite to https://github.com/WordPress/gutenberg/commit/1da9ae65c27fe95a6fb1c2bc0da94d43a0202d16. Keep that entry in the top-level and remove it from all WP packages as it's the same version 😅

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/43511/commits/666a624f93035e995d44cc00481e3366a581381b.